### PR TITLE
Ladybird: Add layout tests

### PR DESCRIPTION
**Pull request by kalenikaliaksandr**

### CODE REVIEW

**Review by linusg — COMMENTED** [review]
These should live under `Tests/LibWeb` IMO, even if Ladybird is the tool run tun them, for now.

File: Ladybird/TestLayout/layout_test.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (linusg) [inline Ladybird/TestLayout/layout_test.sh:1]: Why does this exist twice now?

Comment (kalenikaliaksandr) [reply Ladybird/TestLayout/layout_test.sh]: sorry, messed it up a bit while moving directories. fixed.

**Review by linusg — COMMENTED** [review]
Let's call that second commit `Tests: Add LibWeb layout tests` and we're good to go. Thanks for starting to do this :)

**Comment by kalenikaliaksandr** [comment]
thank you for having a look. we will have to wait until "Test LibWeb layout" finish because I haven't seen testing script to succeed while running in github actions yet :)

**Pull request merged**